packages/integrations/useAxios/index.ts

```diff
@@ -124,7 +125,7 @@ export function useAxios<T = any>(url: string, ...args: any[]) {
   }
```

Comment:
This would force the user into using it as a Promise, see how I did it in [`useFetch`](https://github.com/vueuse/vueuse/blob/main/packages/core/useFetch/index.ts#L519). If we return `then` in the return object the user will have the option to await it or to not